### PR TITLE
Remove unnecessary wrapper from LogoButton component

### PR DESCRIPTION
**Pull request by JustCaptcha**
image

### CODE REVIEW

**Comment by JustCaptcha** [comment]
image

File: universal-login-react/src/ui/UFlow/LogoButton.tsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
 export interface LogoButtonProps extends DashboardProps {
   className?: string;
 }
 
-export const LogoButton = ({className, ...rest}: LogoButtonProps) => {
+export const LogoButton = ({className = 'logo-button', ...rest}: LogoButtonProps) => {
   return (
-    <div className={getStyleForTopLevelComponent(className)}>
+    <div className={useClassFor(className)}>
```
Comment (xitronix) [inline universal-login-react/src/ui/UFlow/LogoButton.tsx:11]: I suppose we don't use useClassFor and className

**Comment by JustCaptcha** [comment]
image

**Comment by JustCaptcha** [comment]
image

File: universal-login-react/src/ui/UFlow/LogoButton.tsx [hunk 2]
```diff
@@ -1,14 +1,9 @@
...
 
-export interface LogoButtonProps extends DashboardProps {
-  className?: string;
-}
-
-export const LogoButton = ({className, ...rest}: LogoButtonProps) => {
+export const LogoButton = ({...rest}: DashboardProps) => {
   return (
-    <div className={getStyleForTopLevelComponent(className)}>
+    <div className='logo-button'>
```
Comment (xitronix) [inline universal-login-react/src/ui/UFlow/LogoButton.tsx:6]: Do we use it?

**Comment by JustCaptcha** [comment]
image

**Pull request merged**